docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po

```diff
@@ -299,44 +299,62 @@ msgstr "폴더 삭제하기"
 #: ../../vfolder/vfolder.rst:143
 msgid ""
 "If you have permission to delete the storage folder, you can delete it by"
-" clicking the trash can icon in the Control panel. When you click the "
-"Delete button, a confirm dialog appears. To prevent accidental deletion, "
-"you have to enter the name of the folder to be deleted, explicitly."
+" clicking the 'trash can' icon in the control column. By clicking the "
+"icon, the system will automatically move the folder to 'trash can' with "
+"following notification,"
+msgstr ""
+"Storage 폴더를 삭제할 수 있는 권한을 가지고 있는 경우, 'Control' 열의 휴지통 아이콘을 눌러 폴더를 삭제할 수 "
+"있습니다. 휴지통 버튼을 클릭하면 안내 메시지가 나타나며 자동으로 폴더가 휴지통으로 이동하고, "
+
+#: ../../vfolder/vfolder.rst:-1
+msgid "Folder delete notification modal"
+msgstr ""
+
+#: ../../vfolder/vfolder.rst:152
+msgid ""
+"and the folder status appearing in the 'status' column will be changed to"
+" ``delete-pending``."
+msgstr "'status' 열에 나타나는 폴더 상태가 'delete-pending' 으로 바뀌게 됩니다."
+
+#: ../../vfolder/vfolder.rst:-1
+msgid "Trash bin table"
 msgstr ""
-"저장소 폴더 삭제 권한이 있는 경우 제어판에서 휴지통 아이콘을 클릭하여 삭제할 수 있습니다. 삭제 버튼을 클릭하면 확인 대화 상자가"
-" 나타납니다. 실수로 삭제되는 것을 방지하기 위해 삭제할 폴더의 이름을 명시적으로 입력해야합니다."
 
-#: ../../vfolder/vfolder.rst:149
+#: ../../vfolder/vfolder.rst:159
 msgid ""
-"When delete operation starts, It may take some time in order to delete "
-"the target folder completely from storage host. Meanwhile, in order to "
-"block unintended operation such as accessing any file/directory inside "
-"the folder on-delete status, we disable any accessing operations."
+"In this status, you can restore the folder by clicking restore button in "
+"Control pane. If you want to permanently delete the folder, please click "
+"'red trash can' icon in the same column. A confirmation modal will pop up"
+" with an input field saying 'Type folder name to delete'. Make sure you "
+"type the exact folder name correctly into the field, and click the red "
+"``DELETE FOREVER`` button to permanently delete the folder."
 msgstr ""
-"삭제 작업이 시작되고 나서도, 실제 저장된 공간에서 대상 폴더를 삭제하는 데 시간이 다소 소요될 수 있습니다. 삭제중인 상태의 "
-"폴더에 접근하려 하는 것과 같은 의도치 않은 작업을 막기 위해, 폴더에 접근할 수 있는 액션은 모두 비활성화됩니다. "
+"'delete-pending' 상태에서는 Control 열의 '복원' 아이콘을 클릭해 폴더를 복원할 수도 있습니다. 폴더를 완전히 "
```

Comment:
```suggestion
"'delete-pending' 상태에서는 Control 열의 '복원' 버튼을 클릭해 폴더를 복원할 수 있습니다. 폴더를 완전히 "
```

Reply:
https://github.com/lablup/backend.ai-docs-webui/pull/64#issuecomment-2149091268 Under suggestion of @YEONFEEL96 I think we should stick with "icon" instead of "button".